app/components/labs-map.js

```diff
@@ -0,0 +1,120 @@
+import mapboxGlMap from 'ember-mapbox-gl/components/mapbox-gl';
+import { assign } from '@ember/polyfills';
+import { get } from '@ember/object';
+import { computed, action } from '@ember/object';
+import layout from '../templates/components/labs-map';
+
+const highlightedCircleFeatureLayer = {
```

Comment:
Can you remind me why you need this new file? Is this "extending" the labs-map component that's coming from the addon?